cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,97 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to Load other Bundles
+====================================================
+
+.. versionadded:: 2.8
+    Support for bundle dependencies was introduced in Symfony 2.8.
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inheriting from them.
+
+While Composer takes care about making sure these dependencies are loaded,
+you'll also need to enable these bundles in the kernel.
+
+If your bundle is meant to be reused, bundle dependencies will complicate
+your installation documentation. This makes installation and upgrading your
+bundle more tedious for your users.
+
+You can avoid this by specifying your dependencies. This will make sure that
+they are loaded in the kernel. It'll also make sure they are loaded *before*
+your own bundle, to make sure you can extend them.
+
+Additional use case for this is for distribution bundle use cases where one
+bundle is in fact bundling several others.
+
+Specifying dependencies
```

Comment:
"Specifying Dependencies" (title case rules are to capitialize every word, except from closed-class words)